docs/core/setup/index.md

```diff
@@ -0,0 +1,453 @@
+---
+title: Install .NET Core on Windows, Linux, and macOS
+description: Learn where and what to install for .NET Core on Windows, Linux, and macOS versions. Discover the dependencies required to develop, deploy, and run .NET Core apps.
+author: leecow
+ms.author: leecow
+ms.date: 10/30/2019
+zone_pivot_groups: operating-systems-set-one
+---
+
+# Install .NET Core
+
+This article provides information on downloading .NET Core and what is required to install .NET Core. .NET Core can be installed on Windows, Linux, and macOS. You can code in your favorite text editor and produce cross-platform libraries and apps.
+
+.NET Core is made up of two components, the runtime and the SDK. The **runtime** is required to run a .NET Core app, when that app doesn't include the runtime. The **SDK** is required to build .NET Core apps and libraries.
+
+There are different ways to develop .NET Core apps:
```

Comment:
Isn't this too early to introduce? That's one of the reasons why for .NET Framework we have a install for developers and an install for system admins. Two different audiences.

Perhaps here you could add am H2 after prereqs where you say 
## Set up your development machine

And then you start talking about dev environments and whatnot